app/femr/data/models/mysql/RankedPatientMatch.java

```diff
@@ -7,7 +7,11 @@
 @Sql
 public class RankedPatientMatch implements IRankedPatientMatch {
 
-    @OneToOne
+    @OneToOne //(fetch = FetchType.EAGER)
```

Comment:
remove these comments

Reply:
Ya this is a work in progress. If you look at patientEncounter, one of its member variables is patients. I was hoping what worked in patientEncounter would also work with RankedPatientMatch class.